Tasks/DotNetCoreInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -2,11 +2,16 @@
   "loc.friendlyName": ".NET Core SDK Installer",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=853651)",
   "loc.description": "Acquires a specific version of the .NET Core SDK from internet or the local cache and adds it to the PATH. Use this task to change the version of .NET Core used in subsequent tasks.",
-  "loc.instanceNameFormat": "Use .NET Core $(packageType) $(version)",
+  "loc.instanceNameFormat": "Use .NET Core $(packageType)",
+  "loc.group.displayName.generalAdvanced": "Advanced",
   "loc.input.label.packageType": "Package to install",
   "loc.input.help.packageType": "Please select whether to install only runtime or full SDK.",
+  "loc.input.label.useGlobalJson": "use global.json",
+  "loc.input.help.useGlobalJson": "If you set this box, the task will search for all global.json from your working directory and install the matching SDK.",
```

Comment:
`SDK's`

Reply:
done.

Reply:
I think we have resolve this. If not reopen this conversion.